tests/meta/report/conftest.py

```diff
@@ -79,7 +79,7 @@ def get_latest_metadata(self, family_id):
         # Define output dict
         out_data = {
             "analysis_sex": {"ADM1": "female", "ADM2": "female", "ADM3": "female"},
-            "case": "yellowhog",
+            "family": "yellowhog",
```

Comment:
Why the switch back to family again?

Reply:
i think this was data from chanjo or something like that, I guess it felt like a little out of scope